data/Tonga/Assembly/ep-popolo-v1.0.json

```diff
@@ -904,17 +995,12 @@
       ]
     },
     {
-      "area_id": "area/vava'u_no._16",
```

Comment:
I think you're aware of this anyway, but for completeness, and so we don't overlook it again later, we should be careful not to lose this constituency info.